app/test/e2e/dataset-find-by-ids.spec.js

```diff
@@ -0,0 +1,78 @@
+const nock = require('nock');
+const Dataset = require('models/dataset.model');
+
+const { createDataset, deserializeDataset } = require('./utils');
+const { getTestServer } = require('./test-server');
+
+const requester = getTestServer();
+
+let cartoFakeDataset;
+let jsonFakeDataset;
+
+const BASE_URL = '/api/v1/dataset';
+
+describe('Find by ids datasets', () => {
+
+    before(async () => {
+        if (process.env.NODE_ENV !== 'test') {
+            throw Error(`Running the test suite with NODE_ENV ${process.env.NODE_ENV} may result in permanent data loss. Please use NODE_ENV=test.`);
+        }
+
+        nock.cleanAll();
+
+        Dataset.remove({}).exec();
+
+        cartoFakeDataset = await new Dataset(createDataset('cartodb')).save();
+        jsonFakeDataset = await new Dataset(createDataset('json')).save();
+    });
+    
+    it('Return no data if there\'s no such id', async () => {
+        const response = await requester.post(`${BASE_URL}/find-by-ids`)
+            .send({
+                ids: ['non-existing-id']
+            });
+    
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(0);
+    });
+
+    it('Return datasets when getting them by ids', async () => {
+        const ids = [cartoFakeDataset.id, jsonFakeDataset.id];
+        const response = await requester.post(`${BASE_URL}/find-by-ids`).send({ ids });
+
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(2);
+            
+        const datasetsIds = deserializeDataset(response).map(_ => _.id);
+        
+        datasetsIds.should.contain(cartoFakeDataset.id);
+        datasetsIds.should.contain(jsonFakeDataset.id);
+    });
+    
+    it('Return only existing datasets', async () => {
+        const ids = [cartoFakeDataset.id, 'non-existing-id'];
+        const response = await requester.post(`${BASE_URL}/find-by-ids`).send({ ids });
+
+        response.status.should.equal(200);
+        response.body.should.have.property('data').with.lengthOf(1);
+
+        response.body.data[0].id.should.equal(cartoFakeDataset.id);
```

Comment:
please test the response body properly. You don't need to cover every single field, but testing just the IDs is not very helpful to prevent, for example, a bug in a serializer